### PR TITLE
Mttdoc 236 networking basics   clientside interpolation and buffering

### PR DESCRIPTION
**Pull request by Briancoughlin**
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
This Pr adds a new section on Clientside Interpolation from the EDU team [PR#8](https://github.com/Unity-Technologies/com.unity.multiplayer.rfds/pull/8)

Added two new entries to Network Latency management glossary page

**Issue Number:** 

Work being tracked in MTTDOC-236

### CODE REVIEW

File: docs/learn/clientside-interpolation-buffering.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Not only does this  approach cause some unresponsiveness (which may be acceptable in certain game genres,) it also makes it more difficult to aim at other players - the non-up-to-date rendering of the world forces the player to aim ahead of their target to compensate for lag. Worst case scenario - they could be legitimately aiming at the enemy player, but due to the fact that the enemy was actually a 100-150ms "ahead" (as in forward in time, not neccessarily ahead in a positional sense) of what is being rendered - they have no chance to hit the enemy unless he just runs predictably in a straight line. 
+
+Now throw in some chaos by means of [RTT fluctuations](lagandpacketloss#round-trip-time-rtt) / [Jitter](lagandpacketloss#jitter) and we're getting into an unacceptable gameplay experience..
```
Comment (lkrell) [inline docs/learn/clientside-interpolation-buffering.md:18]:
```suggestion
Not only does this approach cause some unresponsiveness (which may be acceptable in certain game genres), it also makes it more difficult to aim at other players. The non-up-to-date rendering of the world forces the player to aim ahead of their target to compensate for lag. As a worst case scenario, the player could be legitimately aiming at the enemy player, but due to the fact that the enemy was actually a 100-150ms "ahead" (as in forward in time, not necessarily ahead in a positional sense) of what is being rendered, they may be unable to hit the enemy unless he runs predictably in a straight line. 

Now throw in some chaos by means of [RTT fluctuations](lagandpacketloss#round-trip-time-rtt) and [Jitter](lagandpacketloss#jitter), and we are getting into an unacceptable gameplay experience.
```

**Review by lkrell — COMMENTED** [review]
Tossed in some feedback, general cleanup, tried to make sure to enter updates in both versions.

File: docs/learn/clientside-interpolation-buffering.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+id: clientsideinterpolationbuffering
```
[the remark this answers is not in the document]
Comment (Briancoughlin) [reply docs/learn/clientside-interpolation-buffering.md:2]: ill change it

File: docs/learn/clientside-interpolation-buffering.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+# Boss Room Example 
+
+
+This technique is implemented in the (BossRoom sample)[https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/]. A brief implementation description is as follows (with further documentation available on the BossRoom GitHub page):
+
+ - all players and Imps are comprised of two logically tied GameObjects:
+    - the first one is updating it's position to match the one sent from the server as soon as we recieve new state
+    - the second one smoothly follows the first GameObject
```
[the remark this answers is not in the document]
Comment (Briancoughlin) [reply docs/learn/clientside-interpolation-buffering.md]: I have asked for this in the ticket that's out for review.

**Review by Cosmin-B — APPROVED** [review]

File: docs/learn/clientside-interpolation.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+---
+id: clientside_interpolation
+title:  Clientside Interpolation
+description: Tutorial that covers the basics of lag mitigation and a way to produce smooth gameplay.
```
Comment (lkrell) [inline docs/learn/clientside-interpolation.md:4]: Tutorial may need edit?

File: docs/learn/clientside-interpolation.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+As we discussed in [Lag and Packet Loss](lagandpacketloss.md) - latency is our enemy, with [Jitter](lagandpacketloss#jitter) being the chaotic element that adds complexity to our task of producing "smooth" gameplay experience.
+
+If we just accept the fact that latency exists, but chose not to do anything about it - we would implement what is known as "dumb terminal". Dumb Terminals don't need to understand anything about the simulation they are visualizing for the client - all they do is:
```
Comment (lkrell) [inline docs/learn/clientside-interpolation.md:9]: Don't to do not
```suggestion
If we just accept the fact that latency exists, but chose not to do anything about it - we would implement what is known as "dumb terminal". Dumb Terminals do not need to understand anything about the simulation they are visualizing for the client - all they do is:
```

**Review by lkrell — APPROVED** [review]
Teeny change and thinking long term on learn content. Awesome!

**Review by chrispope — APPROVED** [review]

File: docs/learn/clientside-interpolation.md (hunk 4 above)
Comment (Briancoughlin) [reply docs/learn/clientside-interpolation.md:4]: changed to guide

**Pull request merged**